src/app/components/profile-membership/profile-membership.component.html

```diff
@@ -1,52 +1,67 @@
+                    <option value="false">Rider</option>
+                </select>
+            </div>
+        </div>
+        <div class="form-group row">
+            <div class="col-sm-1"></div>
+            <label for="select-status" class="col-sm-4 col-form-label">Status:</label>
+            <div class="col-sm-4">
+                <select class="form-control mySelect" name="status" id="select-status" [(ngModel)]="profileObject.active">
+                    <option value="true">Active</option>
+                    <option value="false">Disabled</option>
+                </select>
+            </div>
+        </div>
+        <div *ngIf='carObject' >
             <div class="form-group row">
-            
-                    <div class="col-sm-12">
-                       <h5>Batch:</h5>
-                       <span> {{profileObject.batch.batchNumber}} - {{profileObject.batch.batchLocation}}</span>
-                    </div>
-                </div>
-                
-                <div class="form-group row">
-                     <div class="col-sm-4 t">
-                    <h6>Status:</h6>
-                    <select class="form-control mySelect" name="rider" id="rider" [(ngModel)]=isDriver value="{{profileObject.isDriver}}">
-                        <option value="true">Driver</option>
-                        <option value="false">Rider</option>
-                    </select>
-                    <select class="form-control mySelect" name="active" id="active" [(ngModel)]=active value="{{profileObject.active}}">
-                        <option value="true">Active</option>
-                        <option value="false">Disabled</option>
-                    </select>
-                    <!-- <div class="btn-group" role="group" aria-label="Basic example">
-                        <button type="button" class="btn btn-outline-success mybtngroup">Rider</button>
-                        <button type="button" class="btn btn-outline-success mybtngroup">Driver</button>
-                        <button type="button" class="btn btn-outline-danger mybtngroup">Off</button>
-                    </div> -->
-            
-                <button type="submit" class="btn btn-primary myBtn">Save</button>
+                <div class="col-sm-1"></div>
+                <label for="seats-available-input" class="col-sm-4 col-form-label">Seats Available:</label>
+                <div class="col-sm-4">
+                    <input class="form-control" type="number" name="seats-available" id="seats-available-input" [(ngModel)]="carObject.seatsAvailable"  [disabled]="profileObject.active == 'false' || profileObject.active == false">
                 </div>
             </div>
         </div>
     </div>
-            
+    <button type="submit" class="btn btn-primary myBtn" (click)='updatesMembershipInfo()'>Save</button>
+</div>
+
+<!-- should the following code be deleted? It is not currently in use -->
```

Comment:
If it's been commented out, its safe to remove

Reply:
cool, will remove and push again